src/components/PageLayout/PageLayout.stories.mdx

```diff
@@ -0,0 +1,73 @@
+import { Meta, ArgsTable, Canvas, Story } from '@storybook/addon-docs/blocks';
+import CodeSnippet from '../CodeSnippet/CodeSnippet';
+import Spinner from '../Spinner/Spinner';
+import PageLayout from './PageLayout';
+
+
+<Meta 
+  title="PageLayout"
+  component={PageLayout}
+/>
+
+export const Template = (args) => <PageLayout {...args}>{args.children}</PageLayout>;
+
+### PageLayout
+
+This is a PageLayout component. It takes a title as a prop and an optional children can be passed in.
```

Comment:
We should explain when to use this component (i.e. login pages) and what it does.

Reply:
I think we should rename it to LoginPageLayout component, and if in future we need to use it outside of just the LoginFlow, we can name it aptly then.
Naming it LoginPageLayout helps define the scope for this component and it's recommended usages.